src/app/teams/teams.component.html

```diff
@@ -53,7 +53,7 @@ <h3>
               Request pending
             </button>
           </ng-container>
-          <button mat-raised-button color="primary" [routerLink]="['view', element.doc._id]" i18n>
+          <button mat-raised-button color="primary" [routerLink]="['view', element.doc._id]" *ngIf="element.userStatus!=='requesting'" i18n>
             { element.userStatus, select, member {Manage Team} unrelated {View Team} }
```

Comment:
In this line we should use the `other` keyword to set **View Team** as the default if the user is not a member of the team, instead of only setting the button if `userStatus` is `unrelated`.

They don't write that in the clearest way, but you can see some examples in the [i18n documentation](https://angular.io/guide/i18n#regular-expressions-for-plurals-and-selections)